docs/HTML.md

```diff
@@ -0,0 +1,93 @@
+### 1. `<header>` -- Header 
+* *type:* block-level
+* *content:* Flow content/palpable content
+* *support:* all browsers
+* *example:* 
+```
+ <header> Logo </header>
+```
+<header> Logo </header>
+
+### 2. `<hr>` -- Horizontal Rule
+* *type:* block-level
+* *content:* Flow Content
+* *support:* all browsers
+* *example:*
+``` <p> Some text </p>
+ <hr>
+<p> some other text </p> ```
+<p> some Text </p>
+<hr>
+<p> some other text </p>
+
+### 3. `<div>` -- Document Division
+* *type:* block-level
+* *content:* Flow content/palpable content
+* *support:*  all browsers
+* *example:*
+```<div> <p> Div Example </p> </div> ```
+<div> </p> Div Example </p> </div>
+
+### 4. `<address>` -- Address Element
+* *type:* block-level
+* *content:* flow content/palpable cotent
+* *support:* all browsers
+* *example:*
+```<address> 5555 Hugh Gerection Street <br> Somewhere, FL 55555 </adress>```
+<address> 5555 Hugh Gerection Street <br> Somewhere, FL 55555</address>
+
+### 5. `<ul>` -- Unordered List
+* *type:* block-level
+* *content:* flow content
+* *support:* all browsers
+* *example:*
+```<ul> <li> Thing One </li> <li> Thing Two </li> <li> Thing Three </li> </ul>```
+<ul> <li> Thing One </li> <li> Thing Two </li> <li> Thing Three </li> </ul>
+
+### 6. `<ol>` -- Ordered List
+* *type:* block-level
+* *content:*  flow content/palpable content
+* *support:*  all browsers
+* *example:*
+```<ol> <li> One Thing </li> <li> Two Thing </li> <li> Three Thing </li> </ol>```
+<ol> <li> One Thing </li> <li> Two Thing </li> <li> Three Thing </li> </ol>
+
```

Comment:
This is a good place to document the `<li>` tag, since it's related to the previous two...